### ISSUE TITLE
Log level does not  follow the defination in openchain.yaml

### ISSUE THREAD

**Issue by samding01** [hyperledger-archives/fabric#802]
The latest master code (commit ee4056e5f799527fefc285e9e70d54ee7c543d71) does not follow log level definition in `openchain.yaml`.

Here is part of definition of the `openchain.yaml`:

```
peer:      error
crypto:    error
status:    warning
stop:      warning
login:     error
vm:        warning
chaincode: warning
```

However, after starting 4 peers with security enabled, consensus=obcpbft,  the container (vp0-3) outputs:

```
  ...
 14:47:59.484 [consensus/handler] HandleMessage -> DEBU 0c2 Did not handle message of type DISC_GET_PEERS, passing on to next MessageHandler
 14:47:59.484 [peer] HandleMessage -> DEBU 0c3 Handling OpenchainMessage of type: DISC_GET_PEERS
 14:47:59.484 [peer] beforeGetPeers -> DEBU 0c4 Sending back DISC_PEERS
 14:47:59.484 [peer] SendMessage -> DEBU 0c5 Sending message to stream of type: DISC_PEERS
  ...
```

If run 1 peer in [`sandbox`](https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md), it follows the log level definition, but outputs

```
 block 2d 2d 2d 2d 2d 42 4 .....
```

**Comment by 2016Nishi** [comment]
Found similar issue when testing for consensus on 4 peers.
Is this a duplicate of #763?

**Comment by juliancarrivick-ibm** [comment]
I am happy to have a look at this. I am assuming that `openchain.yaml` has become `peer/core.yaml`.

Glancing at it, it seems the logging levels in the example `core.yaml` do not necessarily correspond with the commands available (e.g. vm is not used at this point in time). Should they only correspond with the top level commands (e.g. node, network, chaincode) or are they intended to be able to be nested (e.g. node status).

Finally, INFO seems to be the default logging level. This seems a bit verbose. Thoughts?

**Comment by juliancarrivick-ibm** [comment]
This was due to it looking for `logging.node` in `core.yaml` which didn't exist at the time of this issue. PR #2076 fixed this and this can now be closed.

**Comment by JonathanLevi** [comment]
Closed by #2076 as per the above note. Thank you @bcbrock, @srderson and of course @juliancarrivick-ibm.
